cli/cmd/env/set.go

```diff
@@ -11,23 +11,21 @@ import (
 
 // SetCmd is sub command for env. Sets the variable for a project
 func SetCmd(ch *cmdutil.Helper) *cobra.Command {
-	var projectPath, projectName, environment string
+	var projectPath, projectName, environment, keyValPair string
 
 	setCmd := &cobra.Command{
-		Use:   "set <key> <value>",
-		Args:  cobra.ExactArgs(2),
+		Use:   "set [<project-name>] [--env=key=value]",
```

Comment:
1. Did you consider `set [<project>] <key> <value>`, similar to e.g. `rill project describe` ([link](https://github.com/rilldata/rill/blob/cf0bcf4dbd4806268f90a0c9fa071ba186bbb472/cli/cmd/project/describe.go#L20-L32))? Not necessarily saying that's better, just curious which approach you think is more idiomatic? (IMO neither mixed required/optional positional args nor required flags feel intuitive, but can't think of other alternatives.)
2. If you go ahead with this approach, note there are some non-generated docs that need to be updated (search `docs/` for `rill env set`)

Reply:
Yeah I see what you mean, I think I will stub it out and try

Reply:
Normalized on 1.